tests/lib/assertions/test-snapd-core22-required-vset.json

```diff
@@ -0,0 +1,16 @@
+{
+  "type": "validation-set",
+  "series": "16",
+  "account-id": "test-snapd",
+  "authority-id": "test-snapd",
+  "name": "test-snapd-core22-required-vset",
+  "sequence": "1",
+  "timestamp": "2023-10-24T19:55:00+00:00",
+  "snaps": [
+    {
+      "id": "amcUKQILKXHHTlmSa7NMdnXSx02dNeeT",
+      "name": "core22",
```

Comment:
Good idea. Note that I will also have to add bluez to the new model, since we do not currently allow a required snap in a validation set that isn't in the model.